server/konnectors/meetup.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request').defaults({ jar: true });
+const cheerio = require('cheerio');
+const ical = require('./ical_feed');
+
+const baseKonnector = require('../lib/base_konnector');
+
+/**
+ * The goal of this konnector is to fetch the iCal of the user with his Meetup events
+ */
+module.exports = baseKonnector.createNew({
+  name: 'Meetup',
+  vendorLink: 'http://www.meetup.com',
+
+  fields: {
+    login: 'text',
+    password: 'password'
+  },
+
+  models: [],
```

Comment:
Yep, so:

* add `Event = require (../models/event)`
* modify this line to `models: [Event]

See https://github.com/cozy-labs/konnectors/wiki/Write-your-first-connector#konnector-definition